spirekey/src/app/(wallet)/accounts/[raccount]/devices/[cid]/page.tsx

```diff
@@ -0,0 +1,41 @@
+'use client';
+import { AccountOverviewSection } from '@/components/AccountOverviewSection/AccountOverviewSection';
+import { AccountTabs } from '@/components/AccountTabs/AccountTabs';
+import DeviceIcons from '@/components/Card/DeviceIcons';
+import { useAccounts } from '@/resolvers/accounts';
+import { Stack } from '@kadena/kode-ui';
+import { useParams } from 'next/navigation';
+import React from 'react';
+import { overviewWrapperClass } from './style.css';
+
+export default function AccountPage() {
+  const params = useParams();
+  const { accounts } = useAccounts();
+  const raccount = decodeURIComponent(params.raccount.toString());
+  const cid = decodeURIComponent(params.cid.toString());
+  const account = accounts?.find((a) => a.accountName === raccount);
+  const device = account?.devices.find((d) => d['credential-id'] === cid);
```

Comment:
I think we should give some more thought to how to show the devices. I don't think it necessary makes sense to drill down to a specific device any longer and the account page should probably show a summary/overview of devices somehow.